user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -306,6 +306,57 @@ export const main = applyHttpMiddlewares(handler, {
 });
 ```
 
+### Generate mock event with the contract
+
+With the `getMockHandlerInput` feature, you can quickly generate input events to test your lambda functions.
+Using faker `json-schema-faker`, it will generate random data based on the contract schemas and allow you overriding certain key with your own values.
+
+```ts
+import { getMockHandlerInput } from '@swarmion/serverless-contracts';
+
+const [mockEvent] = getMockHandlerInput(myContract, {
+  pathParameters: { userId: '15' },
+  queryStringParameters: { testId: 'plop' },
+});
+```
+
+The result value for mockEvent will be something like
+
+```ts
+const mockEvent = {
+  pathParameters: {
+    userId: '15',
+    pageNumber: '42', // Randomly generated
```

Comment:
This may be more on the `json-schema-faker` side, but is there a way to fix the seed for the random generation ?

I think having non-deterministic tests is not a good practice